scripts/workers/submission_worker.py

```diff
@@ -38,8 +39,8 @@
     "DJANGO_SETTINGS_MODULE", "settings.dev"
 )
 DJANGO_SERVER = os.environ.get("DJANGO_SERVER", "localhost")
-LIMIT_CONCURRENT_SUBMISSION_PROCESSING = os.environ.get(
-    "LIMIT_CONCURRENT_SUBMISSION_PROCESSING"
+LIMIT_CONCURRENT_SUBMISSION_PROCESSING = eval(
```

Comment:
Instead of `eval()`, it would definitely be a better idea to do `int()` since this is always going to be integer. By doing `eval()`, you are opening doors for bugs later in the codebase.